docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md

```diff
@@ -21,6 +21,7 @@ Review this preparatory checklist before initiating your go-live plan.
 You cannot successfully deploy a project on Spryker Cloud Commerce OS unless you do the following:
 
 - Upgrade your project's code to match the latest demoshop release, or at minimum, upgrade to a release that fully supports the Docker SDK.
+- Make sure you have updated spryker/twig to at least version 3.15.2 as this version has important stability improvements over version 3.15.1
```

Comment:
```suggestion
- Update spryker/twig to version 3.15.2 or higher because this version and higher have important stability improvements over version 3.15.1.
```